docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -70,7 +70,11 @@ and the user running Oasis Node is `oasis`, you can do:
 sudo adduser oasis sgx
 ```
 
-Failure to do so may result in permission denied errors during runtime startup.
+Then, log out from your session (desktop, ssh) and back in for permissions to
```

Comment:
Maybe say something like "If you are planning to run your node from an interactive session, make sure to log out from your ...". Because if you are running via systemd or similar, no such thing is needed as the process manager initializes the session.